backend/src/main/java/de/christianporsch/backend/model/CardImage.java

```diff
@@ -0,0 +1,15 @@
+package de.christianporsch.backend.model;
```

Comment:
why is this not part of the actual card class? Just beeing curious

Reply:
thats weird. I have updated my model and these datatypes have been updatet :(((